python/lsst/ci/middleware/output_repo_tests.py

```diff
@@ -114,8 +114,8 @@ def check_objects(self, test_case: unittest.TestCase) -> None:
         """Run tests on the objectTable and objectTable_tract datasets.
 
         This includes tests of storage class conversion, since objectTable
-        tract is defined as a DataFrame in its producing and consuming tasks,
-        but ArrowTable in the data repository."""
+        tract is defined as a ArrowAstropy in its producing and consuming
+        tasks, but ArrowTable in the data repository."""
```

Comment:
Just curious why ArrowTable in the repo?

Reply:
I wanted to test a storage class conversion and didn't want to use Pandas, so I just needed something else `ArrowAstropy` was bidirectional convertible with.